migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -166,6 +166,23 @@ In {product-title} 3.11, OpenShift SDN was the default networking plugin in Red
 
 For information on migrating to OVN-Kubernetes from OpenShift SDN, see xref:../networking/ovn_kubernetes_network_provider/migrate-from-openshift-sdn.adoc#migrate-from-openshift-sdn[Migrating from the OpenShift SDN network plugin].
 
+[WARNING]
+====
+It is not possible to upgrade a cluster to {product-title} 4.17 if it is using the OpenShift SDN network plugin. You must migrate to the OVN-Kubernetes plugin before upgrading to {product-title} 4.17.
+====
+
+For information on OVN-Kubernetes features that are similar to features in the OpenShift SDN plugin see:
+
+* xref:../networking/ovn_kubernetes_network_provider/configuring-egress-ips-ovn.adoc#configuring-egress-ips[Configuring an egress IP address]
+
+* xref:../networking/network_security/egress_firewall/configuring-egress-firewall-ovn.adoc#configuring-egress-firewall-ovn[Configuring an egress firewall for a project]
+
+* xref:../networking/ovn_kubernetes_network_provider/enabling-multicast.adoc#enabling-multicast[Enabling multicast for a project]
+
+* xref:../networking/ovn_kubernetes_network_provider/deploying-egress-router-ovn-redirection.adoc#deploying-egress-router-ovn-redirection[Deploying an egress router pod in redirect mode]
+
+* xref:../networking/network_security/network_policy/multitenant-network-policy.adoc#multitenant-network-policy[Configuring multitenant isolation with network policy]
```

Comment:
(and this part in particular is already linked from the "Network isolation mode" section earlier in this doc.)